cobra/flux_analysis/moma.py

```diff
@@ -1,6 +1,70 @@
 from scipy.sparse import dok_matrix
 
 from ..solvers import get_solver_name, solver_dict
+import cobra.util.solver as sutil
+import sympy
+
+
+def moma_model(model):
```

Comment:
Ok, implemented. Also removed a function deprecated in 0.4 and converted the docstrings to numpy format. Have you seen the changes in `test_flux_analysis.py`? I substituted some benchmarks with smaller functions since the whole benchmarking tok 30 min + with cplex and even longer if you also check with scipy (which is sloooow).